lib/shopify_api/resources/assigned_fulfillment_order.rb

```diff
@@ -0,0 +1,12 @@
+module ShopifyAPI
+  class AssignedFulfillmentOrder < Base
+    CANCELLATION_REQUESTED = 'cancellation_requested'
+    FULFILLMENT_REQUESTED = 'fulfillment_requested'
+    FULFILLMENT_ACCEPTED = 'fulfillment_accepted'
+
+    def self.all(options = {})
+      assigned_fulfillment_orders = super(options)
+      assigned_fulfillment_orders.map { |afo| FulfillmentOrder.new(afo.as_json) }
```

Comment:
If we do `AssignedFulfillmentOrder < FulfillmentOrder`, is this still necessary?

Reply:
Seemed like that should work but it results in `uninitialized constant ShopifyAPI::FulfillmentOrder (NameError)`